### ISSUE TITLE
Cannot start game, Auth error in logs

### ISSUE THREAD

**Issue by kodi** [syncore/qclauncher#4]
```JSON
{"level":"error","ts":"2017-08-02T02:59:19.892+0200","msg":"authenticate: unexpected verify response type","error":"User authentication failed","data":null}
{"level":"error","ts":"2017-08-02T02:59:19.892+0200","msg":"main: Error occurred while executing the launch process.","error":"Received an unexpected response during authentication"}
```

**Comment by syncore** [comment]
**Issue**: This error is due to a stale authentication token and will be addressed in the next release as part of Issue #3 when I find the time to work on this.

**Solution**: delete `data.qcl` and re-enter credentials.

**Comment by kodi** [comment]
deleting a file works, thanks!

**Comment by syncore** [comment]
Fixed in v1.03
